src/nvim/ex_docmd.c

```diff
@@ -140,6 +140,36 @@ struct dbg_stuff {
   except_T    *current_exception;
 };
 
+typedef enum {
+  parse_cmd_SUCCESS,
+  parse_cmd_NOOP,  // may be an error, check parse_cmd_T::errormsg
+} parse_cmd_result_T;
```

Comment:
could just use a boolean? There is also `TriState`.

Reply:
Yeah, thought I'd need a lot more than a boolean, turns out not - ditched :+1: